frontend/dockerfile/dockerfile2llb/image.go

```diff
@@ -58,3 +60,16 @@ func clone(src Image) Image {
 	img.Config.Entrypoint = append([]string{}, src.Config.Entrypoint...)
 	return img
 }
+
+func emptyImage() Image {
+	img := Image{
+		Image: ocispec.Image{
+			Architecture: runtime.GOARCH,
+			OS:           runtime.GOOS,
+		},
+	}
+	img.RootFS.Type = "layers"
+	img.Config.WorkingDir = "/"
+	img.Config.Env = []string{"PATH=" + system.DefaultPathEnv}
```

Comment:
Is this intentional?
(LGTM if so)

Reply:
Yes, in current implementation every stage has default path https://github.com/moby/moby/blob/4047cede65862aa0ea5616297d7c0f3b12526ad4/builder/dockerfile/evaluator.go#L223